doc/nrf-bm/app_dev/dfu/memory_partitioning.rst

```diff
@@ -106,3 +106,48 @@ The placement of the SoftDevice in memory is predetermined and must align with s
 - **SD_StartAddr**: Must align on a 2 kB memory boundary as specified in the SoftDevice release notes.
 
 The size of the SoftDevice is adjusted taking into account the above alignment and space reservations.
+
+Partitions sizes reference
+**************************
+
+The following table shows the recommended sizes for each partition when using the Single-Bank Device Firmware Update (DFU) Process.
+
```

Comment:
General comment to the size in the table.
Other places in doc we use `kB` for size, please adjust to use same in table.